pkg/collector/corechecks/servicediscovery/usm/php.go

```diff
@@ -36,7 +36,7 @@ func (p phpDetector) detect(args []string) (ServiceMetadata, bool) {
 	for _, arg := range args {
 		hasFlagPrefix := shouldSkipArg || strings.HasPrefix(arg, "-")
 		includesAssignment := shouldSkipArg || strings.ContainsRune(arg, '=') || strings.HasPrefix(arg, "-d")
-		shouldSkipArg := prevArgIsFlag || hasFlagPrefix || includesAssignment
+		shouldSkipArg = prevArgIsFlag || hasFlagPrefix || includesAssignment
```

Comment:
Unless I'm missing something, if `shouldSkipArg` becomes true, now it will be true for the entire duration of the loop since the previous value is OR'd in to `hasFlagPrefix` and `includesAssignment`. That doesn't look like expected behaviour?